internal/codegen/stencil_test.go

```diff
@@ -121,3 +121,25 @@ func ExampleStencil_PostRun() {
 	// Output:
 	// hello
 }
+
+func TestStencilPostRunError(t *testing.T) {
+	var (
+		name    = "TestStencilPostRunError"
+		command = "invalidPostRunCommand"
+	)
+
+	ctx := context.Background()
+	fs := memfs.New()
+	mf, err := fs.Create("manifest.yaml")
+	assert.NilError(t, err)
+	_, err = fmt.Fprintf(mf, "name: %s\npostRunCommand:\n- command: %s\n", name, command)
+	assert.NilError(t, err)
+	assert.NilError(t, mf.Close())
```

Comment:
suggestion: can we refactor this to be a test helper since this pattern is in at least two other tests, and it makes sense for this to be used in future tests? Something like:

```go
func createFakeModuleWithManifest(
  t *testing.T,
  manifestContents string,
) billy.Filesystem
```

This also gives us the opportunity to `defer` the `mf.Close()` function call, although you maybe would still want to check the error, it depends on what errors `.Close()` gives.

Reply:
Yes, we can create a separate method for this.
We don't need to keep the manifest file open, so we don't really have to defer the `mf.Close()`.

Reply:
> We don't need to keep the manifest file open, so we don't really have to defer the `mf.Close()`.

I think you'll want to `defer` the `.Close()` call in the event that `assert.NilError()` is triggered for your `.Write()` call.